frontend/src/components/worksheets/Worksheet.js

```diff
@@ -949,7 +983,7 @@ const styles = (theme) => ({
         paddingBottom: 25,  // Height of Footer
     },
     worksheetOuter: {
-        maxWidth: 1200, // Worksheet width
+        maxWidth: 1960, // Worksheet width
```

Comment:
Yeah, as @yipenghe noticed, maybe it would be better to keep the width the same and change it in a PR.